packages/ui/feature-builder-form-controls/src/lib/pipes/is-trigger-guard.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { ActionBase, TriggerBase } from '@activepieces/pieces-framework';
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'isTriggerGuard',
+  standalone: true,
+  pure: true,
+})
+export class IsTriggerGuardPipe implements PipeTransform {
+  transform(value: ActionBase | TriggerBase): value is TriggerBase {
+    return (value as TriggerBase).type !== undefined;
```

Comment:
Shouldnt you check for type values? I dont understand

Reply:
yeah you are right that's a clearer way of doing it, will do, it is just that actions don't have types, but triggers do.
"Polling and Instant"